Packs/CommonScripts/Scripts/ServerLogs/ServerLogs.py

```diff
@@ -3,28 +3,46 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
+ERROR_MESSAGE = ("The script could not execute the `ssh` command. Please create an instance of the "
```

Comment:
No need to make this constant - you can just write the message in the error. 
Also please specify it is the v2 (since the v1 is deprecated)